modules/vocabularies/src/main/frontend/src/vocabularies-admin-page.jsx

```diff
@@ -17,10 +17,29 @@
 //  under the License.
 //
 
+import { Box, Grid, Typography } from "@material-ui/core";
 import React from "react";
 
 function VocabulariesAdminPage(props) {
-  return(<div></div>);
+  return (
+    <Grid container direction="column" justify="space-between">
```

Comment:
There's a dedicated `spacing` prop for adding space between grid items:

```suggestion
    <Grid container direction="column" spacing={5} justify="space-between">
```